portality/templates/includes/header.html

```diff
@@ -42,7 +40,7 @@ <h2 class="label">Menu</h2> <span data-feather="menu" aria-hidden="true"></span>
 {% endif %}
 <nav class="secondary-nav {{ extraClass }}">
     <div class="container">
-        <ul class="nav-bar secondary-nav__menu">
```

Comment:
Did not notice when it happened but the `<h2 class="sr-only">Secondary actions</h2>` (or something along those lines) belongs here @amdomanska

Reply:
@sssoz Yep, that's the problem of this merge. It wasn't very straightforward that's why I needed another pair of eyes. Thank you!

Reply:
Interestengly that is not there even on live...

Reply:
Yes, that is why I mentioned that I did not notice when it was removed

Reply:
(Fixed to "Secondary actions") in 58700ee534e236dcd15fe5f67fb66299aca49e51